admin/InitDb.pl

```diff
@@ -238,18 +239,7 @@ sub CreateRelations
     $ENV{'PGPASSWORD'} = $DB->password;
 
     system(sprintf(qq(echo "CREATE SCHEMA %s" | $psql $opts), $_))
-        for (qw(
-            musicbrainz
-            cover_art_archive
-            documentation
-            event_art_archive
-            json_dump
-            report
-            sitemaps
-            statistics
-            wikidocs
-            dbmirror2
-        ));
+        for @FULL_SCHEMA_LIST;
```

Comment:
I'm going to channel @yvanzo and say that this should probably be a separate commit than the new script?